Update description of user privileges on UTL_FILE (#5876)
EnterpriseDB/docs: product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/20_utl_file.mdx

GRANT EXECUTE ON PACKAGE SYS.UTL_FILE TO mary;
```

> Also, for a UTL_FILE function or procedure to process successfully, the following conditions must be true:

Reviewer: ```suggestion
Also, for a `UTL_FILE` function or procedure to process successfully, you need:
```

> - The user must have the `EXECUTE` privilege on the `UTL_FILE` package. Without that privilege the user is not able to invoke any package APIs. 

Reviewer: FYI, better to state in the positive than the negative. Easier to follow.
